docs/architecture/raft.rst

```diff
@@ -65,6 +70,7 @@ When all the nodes in the cluster and updated and restarted, the cluster will st
 Alternatively, you can enable the ``consistent_cluster_management`` option when you are:
 
 * Performing a rolling upgrade from version 5.1 to 5.2 or version 2022.x to 2023.1 by updating ``scylla.yaml`` before restarting each node. The Raft upgrade procedure will start as soon as the last node was upgraded and restarted. As above, this requires :ref:`verifying <verify-raft-procedure>` that the procedure successfully finishes.
+* Performing a rolling upgrade from version 5.4 to 6.0 or version 2024.1 to 2024.2. ``consistent_cluster_management`` is mandatory in 6.0 and 2024.2, so updating ``scylla.yaml`` is unnecessary. As above, the Raft upgrade procedure will start as soon as the last node was upgraded and restarted. This also requires :ref:`verifying <verify-raft-procedure>` that the procedure successfully finishes.
```

Comment:
Maybe we should refrain from referring to non-existing (but predicted) versions in documentation for now?
@annastuchlik do you have some standard process for such issues?

Reply:
Or leave a reference to the "Verifying" section on the Raft page.
Which there already is: https://opensource.docs.scylladb.com/master/upgrade/upgrade-opensource/upgrade-guide-from-5.1-to-5.2/upgrade-guide-from-5.1-to-5.2-generic.html#validate-raft-setup
But I see it's missing in the ->5.4 upgrade guide.